dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/config/SSLConfigManager.java

```diff
@@ -503,6 +514,8 @@ private SSLConfig parseSecureSocketLayer(Map<String, Object> map, boolean reinit
         String sslProtocol = (String) map.get("sslProtocol");
         if (sslProtocol != null && !sslProtocol.isEmpty()) {
             try {
+                Tr.debug(tc, "sslProtocol: " + sslProtocol);
+                //Print stack trace
                 protocolHelper.checkProtocolValueGood(sslProtocol);
                 sslprops.setProperty(Constants.SSLPROP_PROTOCOL, sslProtocol);
```

Comment:
Does this need the same FIPS 140-3 check and forcing to TLS 1.2 and 1.3?

Reply:
The check is done in the checkProtocolValueGood.